### PR TITLE
metrics: Add extra necessary metrics for monitoring SlowTrend performance.

### PR DESCRIPTION
**Pull request by LykxSassinator**
### What is changed and how it works?

Issue Number: Ref #15271 

What's Changed:

```commit-message
This pr supply extra necessary metrics for `SlowTrend`, which is useful for users to monitor the performance of
`SlowTrend`, including:

1. Slow Trend, shows the changing rate of the slowness on I/O operations. 'value > 0' means the related store might has a slow trend.

2. AVG Sampling Latency, records the sampling latency of recent queries. 
A larger value indicates that the store is more likely to be the slowest store. 
Only when one of TiKV node has been marked with **Slow Trend**, and its severity reaches the limitation,
it will be chosen as the slow node.

3. QPS of each store, store slow trend result, showing the QPS of each store.

4. QPS Changing Rate, shows the changing trend of QPS on each store. 'value < 0' means the QPS has a dropping trend.
```
The related metrics are assembled into `Slow Trend Statistics` row:
image

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 

```release-note
None.
```

### CODE REVIEW

**Comment by bufferflies** [comment]
show this grafana view.

**Comment by LykxSassinator** [comment]
> show this grafana view.

Got, updated

**Comment by tonyxuqqi** [comment]
1） QPS of each store： What's the QPS meaning here? Is it dup with other existing metrics?
2) QPS Changing rate: I think qps change itself cannot tell because it can be the workload change.  Is this data used as the supporting metrics to determine if a node is slow or not?

File: metrics/grafana/tikv_details.json [hunk 1]
```diff
@@ -48363,6 +48363,377 @@
...
+          },
+          "type": "graph",
+          "xaxis": {
+            "mode": "time",
+            "show": true,
+            "values": []
+          },
+          "yaxes": [
+            {
+              "format": "decbytes",
```
Comment (tonyxuqqi) [inline metrics/grafana/tikv_details.json:48453]: format is not appropriate

**Review by SpadeA-Tang — APPROVED** [review]

**Comment by LykxSassinator** [comment]
The metrics have been updated as followings shows:
image

**Comment by LykxSassinator** [comment]
Updated as following:
image

**Review by bufferflies — APPROVED** [review]

**Review by tonyxuqqi — APPROVED** [review]

**Pull request merged**